src/include/migraphx/simplify_algebra.hpp

```diff
@@ -14,6 +14,8 @@ struct module;
  */
 struct simplify_algebra
 {
+    // flag to use shortcuts for some math functions
```

Comment:
I think it'll be better to refer to the specific math function we support currently, which is fast_gelu. Then if more are introduced, we can either comment on where to find all of the fast math functions we support or just enumerate them here.